### PR TITLE
Add Criteo adapter

### PR DESCRIPTION
**Pull request by icaroulle**
## Type of change
- [x] New bidder adapter

## Description of change

- test parameters for validating bids
```
{
  bidder: 'criteo',
  params: {
    zoneId: 497747
  }
}
```
- contact email of the adapter’s maintainers: i.caroulle@criteo.com h.duthil@criteo.com
- [x] official adapter submission

### CODE REVIEW

**Comment by matthewlane** [comment]
Thanks for the PR. Are the test parameters still valid? I'm getting back empty bids with them

**Comment by icaroulle** [comment]
Hello Matt,
Thank you for the feedback. This might be a tagging issue and we can recommend you to test a different zone. Can you check with zoneId: 652358 (which is a 300x250) and go on http://www.jcpenney.com/, browse a few items and put some in a basket. You should get bids after trying this.

File: src/adapters/criteo.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  // send bid request to criteo direct bidder handler
+  function _pushBidRequestEvent(params) {
+
+    // if we want to be fully asynchronous, we must first check window.criteo_pubtag in case publishertag.js is not loaded yet.
+    window.Criteo = window.Criteo || {};
+    window.Criteo.events = window.Criteo.events || [];
+
+    // generate the bidding event
+    var biddingEventFunc = function () {
+
```
Comment (mkendall07) [inline src/adapters/criteo.js]: can you remove some of these extra line breaks?

**Comment by mkendall07** [comment]
Getting bids back so going to merge for now. We can address style clean up later.

**Pull request merged**